client/state/billing-transactions/transaction-filters/test/reducer.js

```diff
@@ -0,0 +1,153 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import {
+	BILLING_TRANSACTIONS_FILTER_SET_APP,
+	BILLING_TRANSACTIONS_FILTER_SET_DATE,
+	BILLING_TRANSACTIONS_FILTER_SET_PAGE,
+	BILLING_TRANSACTIONS_FILTER_SET_QUERY,
+} from 'state/action-types';
+import { app, date, page, query } from 'state/billing-transactions/transaction-filters/reducer';
+
+describe( 'transaction filter reducer', () => {
+	describe( '#app', () => {
+		test( 'should update from empty filter', () => {
```

Comment:
We don't seem to have a test that tests the initial default value of the reducer.